src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -31,7 +31,7 @@ import { get } from 'lodash';
 
 export class TransactionApi implements ITransactionApi {
   private static readonly ERROR_ARRAY_PATH = 'nonFieldErrors';
-  private static readonly INDEFINITE_EXPIRATION_TIME = -1;
+  private static readonly MAX_TTL = 2147483647;
```

Comment:
Nit: I'd add a comment explaining this value and add some separators `2_147_483_647` or change it to `2^31 - 1`.